resources/cdk/photo_asset_manager/README.md

```diff
@@ -70,26 +70,26 @@ $Env:PAM_LANG =
 
 There are three stacks that need to be deployed:
 
-1. PamFrontEndInfraStack ({PAM_NAME}-FE-Infra-PAM) - Cloudfront distribution and S3 bucket host for static website files.
-2. PamStack ({PAM_NAME}-{PAM_LANG}-PAM) - The backend resources (Rekognition, Lambda, S3, Cognito, ApiGateway).
+1. PamFrontEndInfraStack ({PAM_NAME}-FE-Infra-PAM) - Amazon Cloudfront distribution and S3 bucket host for static website files.
+2. PamStack ({PAM_NAME}-{PAM_LANG}-PAM) - The backend resources (Amazon Rekognition, Lambda, Amazon S3, Amazon Cognito, API Gateway).
 3. PamFrontEndAssetStack ({PAM_NAME}-FE-Assets-PAM) - The static website assets.
 
-At this point you can now synthesize the CloudFormation template for this code. Run `cdk ls` to see
+At this point you can now synthesize the AWS CloudFormation template for this code. Run `cdk ls` to see
 a list of available stacks to synth/deploy.
 
 ```
 cdk synth {PAM_NAME}-FE-Infra-PAM
 ```
 
-If everything looks good, go ahead and deploy! This step will actually make
-changes to your AWS cloud environment.
+If everything looks good, go ahead and deploy. This step will make
```

Comment:
```suggestion
If everything looks good, proceed with `cdk deploy`. This step will make
```

Reply:
All of these changes direct from our editor.